mobile/android/android-application-manager.ts

```diff
@@ -41,18 +41,10 @@ export class AndroidApplicationManager extends ApplicationManagerBase {
 
 	public startApplication(appIdentifier: string, framework?: string): IFuture<void> {
 		return (() => {
-			let startActivityName = this.getStartPackageActivity(framework);
-			let defaultActivityNames = [startPackageActivityNames[TARGET_FRAMEWORK_IDENTIFIERS.Cordova.toLowerCase()],
-				startPackageActivityNames[TARGET_FRAMEWORK_IDENTIFIERS.NativeScript.toLowerCase()]];
-
-			let startActivityNames = startActivityName ? [startActivityName] : defaultActivityNames;
-
-			_.each(startActivityNames, (activityName: string) => {
-				this.adb.executeShellCommand(["am", "start",
-					"-a", "android.intent.action.MAIN",
-					"-n", `${appIdentifier}/${activityName}`,
-					"-c", "android.intent.category.LAUNCHER"]).wait();
-			});
+			this.adb.executeShellCommand(["monkey",
+				"-p", `${appIdentifier}`,
```

Comment:
I don't think there's a need for the template string here - `appIdentifier` is already of type string